src/OpenConext/ProfileBundle/Resources/views/MyServices/ConsentList/service-information.html.twig

```diff
@@ -2,13 +2,29 @@
 {% if specifiedConsent.releasedAttributes is empty %}
     <p>{{ 'profile.my_services.no_attribute_released'|trans }}</p>
 {% else %}
-    {% for sourceName, releasedAttributes in specifiedConsent.releasedAttributesGroupedBySource  %}
-        {# TODO: the AA sourced attributes are yet to be rendered see: https://www.pivotaltracker.com/story/show/179920711 for details #}
+    {% for sourceName, releasedAttributes in specifiedConsent.getIdPAttributes  %}
         {% if sourceName == 'idp' %}
         <ul class="attributeList">
             {% include '@OpenConextProfile/MyServices/AttributeList/idp.html.twig' %}
         </ul>
         {% endif %}
     {% endfor %}
+
+    {% if specifiedConsent.getAttributeAggregatedAttributes|length > 1 %}
```

Comment:
Really >1? Or >=1?

Reply:
I replaced this with a more semantically correct 'is not empty' test.